index.d.ts

```diff
@@ -1,8 +1,15 @@
+interface Options {
+	type?: 'hex' | 'base64' | 'url-safe';
```

Comment:
Should have a doc comment that describes the different choices and an example block with examples of each (like in the readme).